### PR TITLE
google auth coordinates

### PR DESCRIPTION
**Pull request by ngreen28**
I can't change the coordinates when i use google auth

### CODE REVIEW

**Comment by bennextmitchell** [comment]
where do we get our google username and password from? because if I use my email address it crashes and the console looks like this:

Unhandled Exception: Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: b. Path 'GoogleUsername', line 6, position 21.
   at Newtonsoft.Json.JsonTextReader.ReadStringValue(ReadType readType)
   at Newtonsoft.Json.JsonTextReader.ReadAsString()
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType(JsonReader reader, JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Populate(JsonReader reader, Object target)
   at Newtonsoft.Json.JsonSerializer.PopulateInternal(JsonReader reader, Object target)
   at Newtonsoft.Json.JsonConvert.PopulateObject(String value, Object target, JsonSerializerSettings settings)
   at PoGo.NecroBot.CLI.AuthSettings.Load(String path) in C:\Users\tvgfx\Documents\Pokemon Go Bot\PoGo.NecroBot.Logic\Settings.cs:line 43
   at PoGo.NecroBot.CLI.GlobalSettings.Load(String path) in C:\Users\tvgfx\Documents\Pokemon Go Bot\PoGo.NecroBot.Logic\Settings.cs:line 283
   at PoGo.NecroBot.CLI.Program.Main(String[] args) in C:\Users\tvgfx\Documents\Pokemon Go Bot\PoGo.NecroBot.CLI\Program.cs:line 31

**Comment by bennextmitchell** [comment]
my auth config looks like this,

{
  "AuthType": "google",
  "GoogleRefreshToken": null,
  "PtcUsername": null,
  "PtcPassword": null,
  "GoogleUsername": benm@mitchellsworld.com,
  "GooglePassword": (mypassword)
}

**Comment by kashish4u** [comment]
Hello,
You need to change a bit and add your username and password in inverted commas. like follows to make it work : 

{
  "AuthType": "google",
  "GoogleRefreshToken": "",
  "PtcUsername": null,
  "PtcPassword": null,
  "GoogleUsername": "your@email.com",
  "GooglePassword": "yourpassword",,
}

**Comment by Zashuiba** [comment]
very simple, just put the username and password both between quotes. Just happened to me a few minutes before...

**Comment by bennextmitchell** [comment]
ahh I thought It was something like that but I didn't want to change it just incase. Thanks :p

File: FeroxRev [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit c6d12b6dfbcf51c96457788d6755708373c3d48e
+Subproject commit fb4ad3d87d7502679f03ddcb5fbf34c0bf215f4e
```
Comment (kashish4u) [inline FeroxRev]: Still the login via google authentication token is not working !
Kindly look into it and try to fix it up !
THanks

**Comment by kashish4u** [comment]
Still the bot unable to pick generate google authentication token !!

**Comment by Joshuamtgprice** [comment]
@bennextmitchell 
It'd probably have been much better, if you'd went and made a fresh issue/ticket about this problem, lol, instead of posting it here hidden within someone else's "report". Just came to do the same myself actually, but wanted to make sure noone else had reported this "problem" yet.

@kashish4u Thank you for that solution, it's very much appreciated! =)

**Comment by megarnblove** [comment]
What do i have to write if i use the Trainer Club in the AuthType?

**Comment by zenptg** [comment]
i want to login with PTC account. Help me pls

**Pull request closed without merging**